### DISCUSSION TITLE
Upload file

### DISCUSSION THREAD

**Discussion by Laxy317** [toeverything/blocksuite#1962] [General]
I found `Image or file` here on the menu, but now only images can be uploaded. What types of files will be supported in the future？Will support video?
<img width="327" alt="截屏2023-04-06 16 54 51" src="https://user-images.githubusercontent.com/57893417/230327175-c05bb6f6-28aa-4149-9501-cebd0a1afe77.png">

**Comment by doodlewind** [comment]
We have plan to support more embed types, this requires something more than the editor UI side. Hopefully this could be shipped in our next major release.
